dot/network/service.go

```diff
@@ -49,6 +49,7 @@ const (
 	maxMessageSize = 1024 * 63 // 63kb for now
 
 	gssmrIsMajorSyncMetric = "gossamer/network/is_major_syncing"
+	batchSize              = 100
```

Comment:
this is ok for now but we should probably be propagating txs within a time period that's less than the block time, so maybe we should check how many tx messages we get on average within a 6 second timespan

we also need the txs to be processed within a block time span, otherwise we might end up adding txs to our pool that have already been included, because we ended up validating them too late. maybe we can open another issue for this

Reply:
Yes, I was initially planning to do it time-based. It will require changing the code flow as we need to do the processing async. I will create a separate issue for it.